django/db/backends/base/base.py

```diff
@@ -36,6 +37,8 @@ class BaseDatabaseWrapper(object):
     ops = None
     vendor = 'unknown'
     SchemaEditorClass = None
+    # Classes instantiated inside __init__().
+    validation_class = BaseDatabaseValidation
```

Comment:
I would be okay with including the rest of the class attributes set to `None` (to make it clearer what attributes need to be overridden). But the main reason I didn't include the Base class as a default for the other attributes is that -- unlike `BaseDatabaseValidation` -- I'm not sure the other Base classes are actually valid values (or at least used in any actual backend). Even the dummy backend subclasses those five classes (so the dummy backend is no different from the other backends in this regard).